austation/code/modules/mining/machine_vending.dm

```diff
@@ -0,0 +1,11 @@
+/obj/machinery/mineral/equipment_vendor
+	var/list/ignore_list = list(
+		"Minebot Upgrade: A.I.",
+		"Mining Bot Companion"
+	)
+
+/obj/machinery/mineral/equipment_vendor/Initialize()
+	..()
+	for(var/datum/data/mining_equipment/I in prize_list)
+		if (I.name in ignore_list)
+			prize_list.Cut(I)
```

Comment:
You should use typepaths instead of names

Reply:
no
just literally no.
all of the items in that list have the exact same type path.
they are initialised with a proc that takes their name, item and price as an argument, but otherwise they are all the same class

Reply:
The datums have the same typepath, yeah. But the objects assigned to them don't.
Use `equipment_path` :)

Reply:
I've been totally misunderstanding what you meant and I'm sorry.